modules/index-service/src/main/java/org/opencastproject/index/service/catalog/adapter/CatalogUIAdapterFactory.java

```diff
@@ -120,6 +121,8 @@ public void updated(String pid, Dictionary<String, ?> properties) throws Configu
     final String type = getCfg(properties, CONF_TYPE_KEY);
     Option<String> optCommonMetadata = getOptCfg(properties, CONF_COMMON_METADATA_KEY);
     final boolean isCommonMetadata = optCommonMetadata.isSome() ? Boolean.parseBoolean(optCommonMetadata.get()) : false;
+    Dictionary serviceProperties = new Properties();
+    serviceProperties.put("common-metadata", isCommonMetadata);
```

Comment:
The class has a `common-metadata` constant CONF_COMMON_METADATA_KEY.

Reply:
Good find, thx!